ansible/configs/ansible-multitier-infra/configure_tower.yml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Are the vars in this file meant to be changed by people? If so, please add some comments to identify what they are used for.

If you are just consuming this file inside the config with no expectation that other people will ever need to modify it, fine...but maybe see my other comment.